.github/workflows/test.yml

```diff
@@ -27,15 +27,19 @@ jobs:
           python-version: ${{ env.PYTHON_LATEST }}
       - uses: pre-commit/action@v3.0.1
   test:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-22.04 # Testsuite fails on 24.04 due to issues related to ImageMagick
     needs: lint
     strategy:
       matrix:
         python: ["3.9", "3.10", "3.11", "3.12", "3.13"]
     steps:
       - uses: actions/checkout@v4
-      - name: Install optimizers
+      - name: Install required system packages
         run: |
+          sudo apt-get update -y
+          # For Wand tests
+          sudo apt-get install -y imagemagick
```

Comment:
given 22.04 has imagemagick, do we want this?

Reply:
or.. perhaps try installing `libmagick++-dev libmagickcore-dev libmagickwand-dev` in addition to imagemagick on 24.04?

Reply:
@zerolab I tried to ascertain that the test suite does pass on on 22.04, which it does. That manual install is leftover from trying to fix the 24.04 build.

Not sure why it is failing on 24.04. I know that 24.04 does not ship with ImageMagick installed, so we do that manually. That does not help however, there is something wrong where it fails somewhat silently by returning None.

I'd like to get to the bottom of this, but don't have a lot of capacity to invest at the moment. Perhaps we should pin the workflow to 22.04 for now so the testsuite continues working and create an issue to move to 24.04

Reply:
let's do that. solve the immediate problem, then spend more time looking into 24.04